balancer-js/src/modules/swaps/swaps.module.spec.ts

```diff
@@ -224,7 +224,8 @@ describe('swaps module', () => {
     });
   });
 
-  describe('full flow', () => {
+  describe('full flow', function () {
+    this.timeout(100000);
```

Comment:
Yeah, you're right. That was an optimistic approach, as I don't believe it will actually take that long. I just didn't want tests to fail due to timeout.
I agree with you that we should eventually have a PR to review these tests and identify if it's possible to make them more efficient, so we're able to reduce the timeout without worrying that it will cause the test to fail.